site/static/js/dismiss_banner.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(function() {
```

Comment:
why do we need this

Reply:
I think https://www.docsy.dev/docs/adding-content/shortcodes/#tabbed-panes is enough or what I missed?

Reply:
> why do we need this

Without these js files, the tab switching action may not be possible, I have streamlined the files.

Reply:
> I think https://www.docsy.dev/docs/adding-content/shortcodes/#tabbed-panes is enough or what I missed?

I referenced the [k8s](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingressclass-scope) website as mentioned in #3131

Reply:
can we only add what's required to create tabs e.g. https://learn.netlify.app/en/shortcodes/tabs/
we dont need to copy the k8s theme
I linked k8s as an example to show how tabs were being used in other projects